.github/contributing.md

```diff
@@ -0,0 +1,52 @@
+# Contributing to ForestForesight
+
+This workflow guides contributors on how to address **[confirmed]** issues in alignment with our branching, testing, and styling strategies.
+
+## Table of Contents
+1. [Valid Issues](#valid-issues)
+2. [Workflow](#workflow)
+   - [1. Assign the Issue](#1-assign-the-issue)
+   - [2. Fork the Repository](#2-fork-the-repository)
+   - [3. Develop and Test Your Changes](#3-develop-and-test-your-changes)
+   - [4. Create a Pull Request (PR)](#4-create-a-pull-request-pr)
+   - [5. Review and Respond](#5-review-and-respond)
+   - [6. Approval and Merge](#6-approval-and-merge)
+
+---
+
+## Valid Issues
+Only issues with the label **[confirmed]** should be picked up and fixed. If you are interested in a new feature or bug fix, please check if there is already an issue created. If not, create a new issue, and wait for it to be labeled **[confirmed]** by maintainers before proceeding.
```

Comment:
Not sure if this is the best way to handle it. They can start working on it regardless. but just with the risk that it will not be merged.